File: dotNet/Semantria.Com/Mapping/Output/DocModelSentiment.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;
using Semantria.Com.Mapping.Output;

namespace Semantria.Com.Mapping.Output
{
    [DataContract(Name = "model_sentiment")]
    [XmlRootAttribute("model_sentiment", Namespace = "")]
    public class DocModelSentiment
    {
        [DataMember(Name = "model_name")]
        [XmlElementAttribute("model_name")]
        public string ModelName { get; set; }

        [DataMember(Name = "mixed_score")]
        [XmlElementAttribute("mixed_score")]
        public float MixedScore { get; set; }

        [DataMember(Name = "negative_score")]
        [XmlElementAttribute("negative_score")]
        public float NegativeScore { get; set; }

        [DataMember(Name = "neutral_score")]
        [XmlElementAttribute("neutral_score")]
        public float NeutralScore { get; set; }

        [DataMember(Name = "positive_score")]
        [XmlElementAttribute("positive_score")]
        public float PositiveScore { get; set; }

        [DataMember(Name = "sentiment_polarity")]
        [XmlElementAttribute("sentiment_polarity")]
        public string SentimentPolarity { get; set; }
    }
}

File: Java/src/main/java/com/semantria/mapping/output/DocModelSentiment.java
package com.semantria.mapping.output;

import javax.xml.bind.annotation.XmlElement;

public final class DocModelSentiment
{
    private String model_name = null;
    private Float mixed_score = null;
    private Float negative_score = null;
    private Float neutral_score = null;
    private Float positive_score = null;
    private String sentiment_polarity = null;

    @XmlElement(name="model_name")
    public String getModelName() { return model_name; }
    @XmlElement(name="mixed_score")
    public Float getMixedScore() { return mixed_score; }
    @XmlElement(name="negative_score")
    public Float getNegativeScore() { return negative_score; }
    @XmlElement(name="neutral_score")
    public Float getNeutralScore() { return neutral_score; }
    @XmlElement(name="positive_score")
    public Float getPositiveScore() { return positive_score; }
    @XmlElement(name="sentiment_polarity")
    public String getSentimentPolarity() { return sentiment_polarity; }

    public void setModelName(String model_name) { this.model_name = model_name; }
    public void setMixedScore(Float mixed_score) { this.mixed_score = mixed_score; }
    public void setNegativeScore(Float negative_score) { this.negative_score = negative_score; }
    public void setNeutralScore(Float neutral_score) { this.neutral_score = neutral_score; }
    public void setPositiveScore(Float positive_score) { this.positive_score = positive_score; }
    public void setSentimentPolarity(String sentiment_polarity) { this.sentiment_polarity = sentiment_polarity; }
}
